headers/types/ground_mode/ground_mode.h

```diff
@@ -99,4 +99,15 @@ struct script_object {
 };
 ASSERT_SIZE(struct script_object, 12);
 
+// A global structure holding various pointer to important structure for ground mode
+struct main_ground_data {
+    undefined* script; // 0x0: pointer to script structure
```

Comment:
The formatter likes comments on struct fields to be aligned; you can just download [the formatted file (under "Artifacts")](https://github.com/UsernameFodder/pmdsky-debug/actions/runs/2237612286#artifacts) and apply it to this PR.